### ISSUE TITLE
<Transition appear/> not working 

### ISSUE THREAD

**Issue by StitiFatah** [nuxt/nuxt#22310]
### Environment

- Operating System: Linux
- Node Version:     v16.19.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      devtools, modules
- Runtime Modules:  @nuxtjs/tailwindcss@6.8.0, @vueuse/motion/nuxt@2.0.0
- Build Modules:    -

### Reproduction

https://codesandbox.io/p/sandbox/sleepy-bash-rkcwq7

### Describe the bug

The vue docs says : If you also want to apply a transition on the initial render of a node, you can add the appear prop. 
https://vuejs.org/guide/built-ins/transition.html#transition-on-appear

There is no transition on the initial render with nuxt, it however works when unmounting/remouting the node afterwards. 

Working fine on vanilla vue :  https://codesandbox.io/p/sandbox/jovial-kapitsa-qvhxzt

**Comment by danielroe** [comment]
I think this is a Vue SSR issue - see if you can reproduce without Nuxt with https://stackblitz.com/github/nuxt-contrib/vue3-ssr-starter

**Comment by danielroe** [comment]
Confirmed - see https://stackblitz.com/edit/github-85wcaq?file=src/App.vue. We can track upstream in https://github.com/vuejs/core/issues/6951.

**Comment by liqueflies** [comment]
@danielroe I still see this issue. 
Is there a reason for not planning to resolve it?
Just to understand.

A fairly workaround is to add the same animation at "onMounted" but it's not working at the same mode and we loose the beforeEnter hook.

Thank you so much.

**Comment by danielroe** [comment]
It's not that we don't want to resolve it, but it is a Vue bug and there is an upstream issue tracking it.

**Comment by danielroe** [comment]
Oh, I see it is already meant to be resolved. If you are still encountering it please raise a new issue

**Comment by liqueflies** [comment]
You mean on nuxt repo or vue repo?
I've updated the nuxt dependecy to it's latest version from the demo above:
https://codesandbox.io/p/devbox/nuxt3-appear-4pzthr

Issue still present, if you prefer open a new issue, I'll do that but i think it's the same

**Comment by danielroe** [comment]
Can you reproduce it with Vue without Nuxt?

**Comment by liqueflies** [comment]
> Can you reproduce it with Vue without Nuxt?

If this is the vue-ssr template that I found on nuxt, yes, the problem is the same:

https://stackblitz.com/edit/github-jer1rm

Is this again related to vue?

**Comment by danielroe** [comment]
Yes, if it reproducible there then it should be raised upstream at vuejs/core

**Comment by liqueflies** [comment]
> Yes, if it reproducible there then it should be raised upstream at vuejs/core

Opened the issue into the vuejs/core repository.

Thank you so much.
